docs/dev-tools.rst

```diff
@@ -0,0 +1,15 @@
+Developer Tools
+===============
+
+Python Template
+---------------
+See the file src/python/__template__.py
+
+include:: ../src/python/__template__.py
+
+Bash Script Template
+--------------------
+See the file src/__template__.sh
+
+include:: ../src/__template__.sh
```

Comment:
Fixed in https://github.com/foglamp/FogLAMP/pull/81/commits/0e632b748bcc4a9646cc29c8115f94e82bdffafd